Acme/CertificateAuthority/CertificateAuthorityConfigurationInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace AcmePhp\Bundle\Acme\CertificateAuthority;
```

Comment:
Can you add the header present in other ACME PHP projects? You can add you as an author of the classes if you want :) .